django_fido/views.py

```diff
@@ -43,9 +44,9 @@ class BaseAttestationVerifier(AttestationVerifier):
 
     def ca_lookup(self,
                   attestation_result: AttestationResult,
-                  client_data_hash: AuthenticatorData) -> Optional[List[bytes]]:
+                  client_data_hash: AuthenticatorData) -> Optional[bytes]:
         """Return empty CA lookup to disable trust path verification."""
-        return []
+        return None
```

Comment:
This will potentially break the lookup attestation verification.

Reply:
Returning `None` will throw `UntrustedAttestation`.

Reply:
So would `[]` if we are looking at the same [part](https://github.com/Yubico/python-fido2/blob/main/fido2/attestation/base.py#L235)

Reply:
Ha, I got it confused... the CA verification is broken completely... It is probably a good candidate for redesign/removal.